Assets/HoloToolkit/Input/Scripts/Cursor/Cursor.cs

```diff
@@ -278,16 +273,10 @@ private void TryLoadPointerIfNeeded()
                 // pointer currently registered with FocusManager, we use it.
 
                 Pointer = FocusManager.Instance.TryGetSinglePointer();
-
-                if (Pointer == null)
```

Comment:
Will do.  It looks like the InputManager prefab wasn't updated to use SimpleSinglePointerSelector, so I figure that could be one of the causes, but I'll dig in.